dbt/seeds/ccao/ccao.loa.csv

```diff
@@ -0,0 +1,4945 @@
+year,class_code,loa
+2000,0,0
+2000,100,0.22
```

Comment:
**[Question, non-blocking]** It seems like `ccao.class_dict` records LoA as the percentage value rather than decimal value (i.e. `22` instead of `0.22`). Is it a problem that they're different, or are we intending to move toward decimal as our standard?

Reply:
Good point! I dropped the LoA from `class_dict` in 56ca182